### PR TITLE
Khoi Nguyen

### PR DESCRIPTION
**Pull request by khoifishpond**

### CODE REVIEW

File: lib/event.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+require 'date'
+
+class Event
+  attr_reader :name, :date, :food_trucks
+
+  def initialize(name)
+    @name = name
+    @date = Date.today.to_s #idk how to use dates
```
Comment (mikedao) [inline lib/event.rb:8]: strftime is your friend

File: lib/event.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  def food_truck_names
+    @food_trucks.map do |food_truck|
+      food_truck.name
+    end
+  end
+
+  def food_trucks_that_sell(item)
+    @food_trucks.find_all do |food_truck|
+      food_truck.inventory.include?(item)
```
Comment (mikedao) [inline lib/event.rb:24]: We are reaching too deep. Use check_stock to make your life easier

**Review by mikedao — COMMENTED** [review]
Overall what you have is real good. Look to leverage code you already have written to make your life easier.

Score: Pass

File: lib/event.rb (hunk 2 above)
Comment (khoifishpond) [reply lib/event.rb:24]: Still working on this. Thanks for the feedback!

**Pull request closed without merging**